service-map/src/main/java/fi/nls/oskari/map/geometry/WKTHelper.java

```diff
@@ -181,4 +181,15 @@ public static String getBBOX(final double westBoundLongitude, final double south
         log.debug("BBOX: " + bbox);
         return bbox;
     }
+
+    /**
+     * Get the numeric SRID of an EPSG-code
+     * @param srsName The String representation of the EPSG-code (e.g. EPSG:3857)
+     * @return the (int) id part of the string e.g. 3857
+     */
+     public static int getSRID(String srsName) {
```

Comment:
The ProjectionHelper has similar methods already: https://github.com/oskariorg/oskari-server/blob/master/service-map/src/main/java/fi/nls/oskari/map/geometry/ProjectionHelper.java#L133-L194

We should add this there instead so they are not spread out. Should we check for null param here as well? Maybe throw a ServiceRuntimeException telling that srs was not available instead of the NullPointerException that this would throw?

Reply:
moved and added the runtime exception.